core/src/test/java/org/apache/iceberg/rest/TestRESTCatalog.java

```diff
@@ -2636,7 +2636,11 @@ public void testTableExistsViaHEADRequest() {
             any());
     Mockito.verify(adapter)
         .execute(
-            reqMatcher(HTTPMethod.HEAD, "v1/namespaces/newdb/tables/table", Map.of(), Map.of()),
+            reqMatcher(
+                HTTPMethod.HEAD,
+                "v1/namespaces/" + TABLE.namespace() + "/tables/" + TABLE.name(),
```

Comment:
you should be able to replace this with `ResourcePaths.forCatalogProperties(Map.of()).table(TABLE)`